# --- ROCK-COPYRIGHT-NOTE-BEGIN ---
# 
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# Please add additional copyright information _after_ the line containing
# the ROCK-COPYRIGHT-NOTE-END tag. Otherwise it might get removed by
# the ./scripts/Create-CopyPatch script. Do not edit this copyright text!
# 
# ROCK Linux: rock-src/package/base/wireless-tools/ranlib-and-ar-vars.patch
# ROCK Linux is Copyright (C) 1998 - 2004 Clifford Wolf
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# 
# --- ROCK-COPYRIGHT-NOTE-END ---

--- ./Makefile.orig	2004-06-25 18:54:11.879375160 +0200
+++ ./Makefile	2004-06-25 18:52:36.304904688 +0200
@@ -121,8 +121,8 @@
 # Compilation of the static library
 $(STATIC): $(OBJS)
 	$(RM) $@
-	ar cru $@ $^
-	ranlib $@
+	$(AR) cru $@ $^
+	$(RANLIB) $@
 
 # So crude but so effective ;-)
 # Less crude thanks to many contributions ;-)
